Providers/Scripts/3.x/Scripts/Tests/test_nxProviders.py

```diff
@@ -1976,7 +2072,7 @@ def setUp(self):
         Setup test resources
         """
         self.mykey='MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDLXp6PkCtbpV+P1gwFQWH6Ez0U83uEmS8IGnpeI8Fk8rY/vHOZzZZaxRCw+loyc342qCDIQheMOCNm5Fkevz06q757/oooiLR3yryYGKiKG1IZIiplmtsC95oKrzUSKk60wuI1mbgpMUP5LKi/Tvxes5PmkUtXfimz2qgkeUcPpQIDAQAB'
-        if not os.path.isdir('/home/jojoma') :
+        if os.system('grep jojoma /etc/passwd'):
```

Comment:
It's not a good idea to use /etc/passwd for testing purposes.
- It requires root access
- It affects all the users